### DISCUSSION TITLE
Presto 0.282 Release notes question related to Temporary definition CREATE FUNCTION and DROP FUNCTION

### DISCUSSION THREAD

**Discussion by raqueltoribiosardon** [prestodb/presto#23672] [General] [answered]
Hello,
Prestodb 0.282 release notes state that it fixes Temporary definition CREATE FUNCTION and DROP FUNCTION. 
What does this mean exactly?? What was the problem in presto 0.281. 

Regards

**Comment by imjalpreet** [comment, marked as answer]
@raqueltoribiosardon This PR has some details on the issue: https://github.com/prestodb/presto/pull/19429

**Comment by raqueltoribiosardon** [comment]
Thanks a lot Jalpreet for your explanation!!!!
